lib/elixir_bench_web/controllers/webhooks_controller.ex

```diff
@@ -0,0 +1,62 @@
+defmodule ElixirBenchWeb.WebHooks do
```

Comment:
this isn't general webhooks is it, but specifically github right? I think I'd even put that in a github namespace.